cli/flox-rust-sdk/src/models/container_builder.rs

```diff
@@ -38,3 +48,53 @@ pub enum ContainerBuilderError {
     #[error("failed to stream container to sink")]
     StreamContainer(#[source] std::io::Error),
 }
+
+#[cfg(test)]
+mod tests {
+    use std::fs::{self, File};
+    use std::os::unix::fs::PermissionsExt;
+
+    use indoc::indoc;
+    use tempfile::TempDir;
+
+    use super::*;
+
+    const TEST_BUILDER: &str = indoc! {r#"
+        #!/usr/bin/env bash
```

Comment:
nonblocking: does this ever pick up the system bash?

Reply:
Within the devshell, I don't think so we have a bash in the PATH
Even if it picks the system for some reason, its still just doing an echo, i think the risks associated with that are limited